scripts/set-versions.sh

```diff
@@ -28,31 +29,51 @@ log_and_exit () {
 }
 
 perform_updates () {
-  echo -e "creating release\nsemver version: $SEMVER_VERSION\nversion number: $VERSION_NUMBER"
-
   # update package.json
-  tmp="package.json_temp"
-  jq --arg semverVersion "$SEMVER_VERSION" '.version = $semverVersion' package.json > "$tmp"
-  mv "$tmp" package.json
+  tmp="${PACKAGE_JSON_FILE}_temp"
+  jq --arg semverVersion "$SEMVER_VERSION" '.version = $semverVersion' $PACKAGE_JSON_FILE > "$tmp"
+  mv "$tmp" $PACKAGE_JSON_FILE
+  echo "- $PACKAGE_JSON_FILE updated"
 
 
   # update android/app/build.gradle
-  sed -i -e 's/versionCode [0-9]\+/versionCode '"$VERSION_NUMBER"'/' android/app/build.gradle
-  sed -i -e 's/versionName ".*"/versionName "'"$SEMVER_VERSION"'"/' android/app/build.gradle
+  sed -e 's/\([[:space:]]*versionCode \)[0-9]\+/\1'"$VERSION_NUMBER"'/' -i '' $ANDROID_BUILD_GRADLE_FILE
+  sed -e 's/\([[:space:]]*versionName \)".*"/\1"'"$SEMVER_VERSION"'"/' -i '' $ANDROID_BUILD_GRADLE_FILE
+  echo "- $ANDROID_BUILD_GRADLE_FILE updated"
 
 
   # update bitrise.yml
-  sed -i -e 's/VERSION_NAME: .*/VERSION_NAME: '"$SEMVER_VERSION"'/' bitrise.yml
-  sed -i -e 's/VERSION_NUMBER: [0-9]\+/VERSION_NUMBER: '"$VERSION_NUMBER"'/' bitrise.yml
+  sed -e 's/\([[:space:]]*VERSION_NAME: \).*/\1'"$SEMVER_VERSION"'/' -i '' $BITRISE_YML_FILE
+  sed -e 's/\([[:space:]]*VERSION_NUMBER: \)[0-9]\+/\1'"$VERSION_NUMBER"'/' -i '' $BITRISE_YML_FILE
   # update flask version numbers in bitrise.yml
-  sed -i -e 's/FLASK_VERSION_NAME: .*/FLASK_VERSION_NAME: '"$SEMVER_VERSION"'/' bitrise.yml
-  sed -i -e 's/FLASK_VERSION_NUMBER: [0-9]\+/FLASK_VERSION_NUMBER: '"$VERSION_NUMBER"'/' bitrise.yml
+  sed -e 's/\([[:space:]]*VERSION_NUMBER: \)[0-9]\+/\1'"$VERSION_NUMBER"'/' -i '' $BITRISE_YML_FILE
+  sed -e 's/\([[:space:]]*FLASK_VERSION_NUMBER: \)[0-9]\+/\1'"$VERSION_NUMBER"'/' -i '' $BITRISE_YML_FILE
+  echo "- $BITRISE_YML_FILE updated"
+
 
   # update ios/MetaMask.xcodeproj/project.pbxproj
-  sed -i -e 's/MARKETING_VERSION = .*/MARKETING_VERSION = '"$SEMVER_VERSION;"'/' ios/MetaMask.xcodeproj/project.pbxproj
-  sed -i -e 's/CURRENT_PROJECT_VERSION = [0-9]\+/CURRENT_PROJECT_VERSION = '"$VERSION_NUMBER"'/' ios/MetaMask.xcodeproj/project.pbxproj
+  sed -e 's/\([[:space:]]*MARKETING_VERSION = \).*/\1'"$SEMVER_VERSION;"'/' -i '' $IOS_PROJECT_FILE
+  sed -e 's/\([[:space:]]*CURRENT_PROJECT_VERSION = \)[0-9]\+/\1'"$VERSION_NUMBER"'/' -i '' $IOS_PROJECT_FILE
```

Comment:
```suggestion
  sed -e 's/\([[:space:]]*MARKETING_VERSION = \).*/\1'"$SEMVER_VERSION;"'/' -i '' $IOS_PROJECT_FILE
  sed -e 's/\([[:space:]]*CURRENT_PROJECT_VERSION = \)[0-9]\+/\1'"$VERSION_NUMBER"'/' -i '' $IOS_PROJECT_FILE
```

```suggestion
  sed -Ei 's/(\s*MARKETING_VERSION = ).*/\1'"$SEMVER_VERSION;"'/' $IOS_PROJECT_FILE
  sed -Ei 's/(\s*CURRENT_PROJECT_VERSION = )[0-9]\+/\1'"$VERSION_NUMBER"'/' $IOS_PROJECT_FILE
```

Do we actually want to do `-i ''`? Omitting the suffix _should_ make it automatic with GNU sed. Also a bit more readable with `-E` imo and closer syntax to what we're used to from JS. The `-e`is redundant with a single expression.